WcaOnRails/config/locales/en.yml

```diff
@@ -1020,7 +1020,7 @@ en:
       use_stripe_below_html: "The registration fee has to be paid through Stripe below once registered."
       refund_policy_html: "If your registration is cancelled before %{limit_date_and_time} you will be refunded a %{refund_policy_percent} of your registration fee."
       no_refunds: "Registration fees won't be refunded under any circumstance."
-      on_the_spot_registration: "On the spot registrations will be accepted with a base registration fee of %{on_the_spot_base_entry_fee}."
+      on_the_spot_registration: "On the spot registrations will be accepted with a base registration fee of %{on_the_spot_base_entry_fee} if the competitor limit has not been reached by %{end_of_registration}"
```

Comment:
Could you include a screenshot of what this looks like? I think we tend to call `wca_local_time` before rendering datetimes to users. You can see an example of this here: https://github.com/jfly/worldcubeassociation.org/blob/00b66534978cd467eff5c0ecef0e12971deb2bcf/WcaOnRails/app/views/posts/_post.html.erb#L26

Reply:
![image](https://user-images.githubusercontent.com/10174579/53116537-a5868800-3594-11e9-9f3d-dfa53aabdf39.png)
`... end_of_registration: wca_local_time(@competition.registration_close))`
computer says no

Reply:
also, I'm sorry about the screenshot, I totally forgot!

![image](https://user-images.githubusercontent.com/10174579/53116657-eaaaba00-3594-11e9-9b92-174d0b8909b0.png)